projects/common/lib/components/button/button.component.ts

```diff
@@ -0,0 +1,58 @@
+import { forwardRef,Component, OnInit, ViewChild, EventEmitter, Input, Output, ElementRef } from '@angular/core';
+import { ControlContainer, ControlValueAccessor, NgForm, NG_VALUE_ACCESSOR } from '@angular/forms';
+
+/**
+ * Button Component is a button which can be used across the application to have same button style. 
+ * 
+ * 
+ * @example
+ *       	<common-button [label]="'Remove Spouse'" 
```

Comment:
Instead of doing: `[label]="'Remove Spouse'"` we can do `label='Remove Spouse'.

Reasons:

* Small performance gain. When we take out the [square brackets] we're telling the host component to no longer analyze that value through the change detector.  (As is, I believe Angular will do a check that the string hasn't changed even through it is a string literal).
* Clear to other programmer the value won't change.
* It also is clear to other programmers that this value will never change